pkg/cli/admin/release/info.go

```diff
@@ -301,22 +300,22 @@ func replaceStableSemanticArgs(args []string, semanticArgs map[string]semver.Ver
 	return nil
 }
 
-func replaceClusterSemanticArgs(f kcmdutil.Factory, args []string, semanticArgs map[string]semver.Version) ([]string, error) {
+func getClusterVersion(ctx context.Context, f kcmdutil.Factory) (*configv1.ClusterVersion, error) {
```

Comment:
I'd done this when I expected to use it in `findClusterIncludeConfig`.  But I ended up going a different way there, because I need more than ClusterVersion.  I've dropped my changes to this file in ae4ddeddc -> 0a9024bb1.